plugins/wasm-rust/src/event_stream.rs

```diff
@@ -86,6 +77,24 @@ impl EventStream {
 
         None
     }
+}
+impl EventStream {
```

Comment:
As a well formated code, there should have a newline bewteen two blocks

Reply:
done in wasm-rust/src *  https://github.com/rust-lang/rustfmt/issues/4866